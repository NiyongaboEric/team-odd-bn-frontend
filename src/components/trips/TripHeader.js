import React from 'react';

const tripHeader = () => (
  <div className="header-container">
    <h2>Trip Request</h2>
  </div>
);
export default tripHeader;
