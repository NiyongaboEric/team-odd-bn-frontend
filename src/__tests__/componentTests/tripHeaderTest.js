import React from 'react';
import { shallow } from 'enzyme';
import TripHeader from '../../components/trips/TripHeader';

describe('Verify email test suite', () => {
  test('renders without crashing', () => {
    expect(shallow(<TripHeader />)).toMatchSnapshot();
  });
});
