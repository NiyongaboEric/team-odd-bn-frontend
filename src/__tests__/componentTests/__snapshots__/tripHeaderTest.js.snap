// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify email test suite renders without crashing 1`] = `ShallowWrapper {}`;
